core/consensus/babe/impl/babe_impl.cpp

```diff
@@ -38,7 +38,8 @@ namespace kagome::consensus {
         hasher_{std::move(hasher)},
         timer_{std::move(timer)},
         event_bus_{event_bus},
-        error_channel_{event_bus_.getChannel<event::BabeErrorChannel>()} {
+        error_channel_{event_bus_.getChannel<event::BabeErrorChannel>()},
+        log_{common::createLogger("BABE")} {
```

Comment:
when logger prints a message it simply concatenates "BABE" with message without whitespace. Maybe we should add whitespace or another symbol to enhance readability.

Reply:
Not true. Just checked that